boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -48,13 +49,23 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
         App_Led_TurnOff(bspd_led);
     }
 
-    if (!App_CanRx_BMS_OkStatuses_BmsOk_Get())
+    if (App_CanRx_BMS_Contactors_AirNegative_Get() == CONTACTOR_STATE_OPEN &&
+        App_CanRx_BMS_Contactors_AirPositive_Get() == CONTACTOR_STATE_OPEN)
     {
-        App_SharedRgbLed_TurnRed(bms_led);
+        App_Led_TurnOn(shdn_led);
     }
     else
     {
-        App_SharedRgbLed_TurnOff(bms_led);
+        App_Led_TurnOff(shdn_led);
+    }
+
+    if (App_CanRx_DCM_Vitals_CurrentState_Get() == DCM_DRIVE_STATE)
+    {
+        App_Led_TurnOn(drive_led);
+    }
+    else
+    {
+        App_Led_TurnOff(drive_led);
     }
```

Comment:
SHDN LED = green if AIR- and + are open, off otherwise
Drive LED = green if DCM is in drive state (beside start switch)